runtime/j9vm/jvm.c

```diff
@@ -4399,7 +4399,7 @@ JVM_Available(jint descriptor, jlong* bytes)
 		return JNI_FALSE;
 	}
 
-#if defined(WIN32) && !defined(__IBMC__)
+#if defined(WIN32) && !defined(__IBMC__) && !(defined(__open_xl__) && !defined(__cplusplus))
```

Comment:
In a C source file `!defined(__cplusplus)` is redundant.

Reply:
wondering why we need to do this to begin with. WIN32 definition already excludes the code from being picked up on AIX or zOS, what is the purpose of && with more other conditions?